strings-csharp/SubstringInString/SubstringInString/Benchmarking.cs

```diff
@@ -1,11 +1,27 @@
 ﻿using BenchmarkDotNet.Attributes;
+using System.Text.RegularExpressions;
 
 [MemoryDiagnoser]
 [Orderer(BenchmarkDotNet.Order.SummaryOrderPolicy.FastestToSlowest)]
 public class Benchmarking
 {
-    public string str = new("Lorem".ToCharArray()[0], 5);
+    private const string str = @"Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                               Nullam quis lorem nec lorem imperdiet tempus. Phasellus lorem justo,
+                               consequat ut lorem sit amet, tempor feugiat lorem.
+                               Proin lorem velit, aliquam vel lorem in, consequat lorem nulla.
+                               Sed lorem justo, eleifend eu lorem id, consectetur lorem neque.
+                               Vestibulum lorem mauris, placerat sit amet lorem nec, laoreet lorem nisi.
+                               Sed nec lorem libero. Nullam lorem lorem, ullamcorper quis lorem vitae.";
+
```

Comment:
I would make this a raw string literal, it will make it cleaner

Reply:
Just to clarify, what you have now is a verbatim string, raw string literals begin and end with """.

Reply:
Okay thanks